integration_test/cases/type.exs

```diff
@@ -348,6 +364,7 @@ defmodule Ecto.Integration.TypeTest do
   end
 
   @tag :decimal_type
+  @tag :decimal_type_cast
   test "decimal type" do
```

Comment:
Should we break this test in two? Unless the last two assertions perform casting on select. Then we can call it `test "decimal type cast" do`.

Reply:
renamed